run/result.go

```diff
@@ -10,6 +10,7 @@ import (
 // The functions associated with Result convert raw data into the desired formats for insertion into the status page template.
 type Result struct {
 	RunID         int
+	FullRun       bool
```

Comment:
Did you consider using an enumeration ? 

```
type RunType string

const (
        FullRun RunType = "FullRun" 
        PartialRun RunType =  "PartialRun"
)

```
Then the field can be called 
`RunType RunType`

The boolean makes the code more difficult to read and is more tedious to extend. 
I like this rule very much. 
https://en.wikipedia.org/wiki/Zero_one_infinity_rule